src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportFunctions.java

```diff
@@ -208,23 +208,19 @@ public void replaceTextInFooter(XWPFDocument doc, Map<String, String> replacemen
      * @param variableList
      * @return
      */
-    public String multipleVariable(List<String> variableList) {
-        switch (variableList.size()) {
-            case 0:
-                return "";
-            default:
-                return String.join(", ", variableList);
-        }
+    public String joinWithComma (List<String> variableList) {
+        return String.join(", ", variableList);
     }
 
-    public String multipleVariableAnd(List<String> variableList) {
-        switch (variableList.size()) {
-            case 0:
-                return "";
-            case 1:
-                return variableList.get(0);
-            default:
-                return String.join(", ", variableList.subList(0, (variableList.size() - 1))) + ", and " + variableList.get(variableList.size() - 1);
+    public String joinWithCommaAnd(List<String> variableList) {
+        if (variableList.isEmpty()) {
```

Comment:
minor: I prefer to use a switch on the size just to be clear about which value changes the outcome. Up to you

Reply:
yeah that's fair, i will change it